docs/pages/index.mdx

```diff
@@ -1,175 +1,220 @@
+  forwards user credentials to the Auth Service, and communicates with Teleport
+  instances that enable access to specific resources in your infrastructure.
+- **Teleport SSH Service:** An SSH server implementation that takes advantage of
+  Teleport's short-lived certificates, sophisticated RBAC, session recording,
+  and other features.
+
+### Prerequisites
+
+<Notice type="tip">
+
+You will need the following to deploy a demo Teleport cluster. If your
+environment doesn't meet the prerequisites above, you can get started with
+Teleport by signing up for a [free trial of Teleport Enterprise
+Cloud](https://goteleport.com/signup/).
 
 </Notice>
 
-## Try out Teleport
+- A Linux host with only port `443` open to ingress traffic. You must be able
+  to install and run software on the host. Either configure access to the host
+  via SSH for the initial setup (and open an SSH port in addition port `443`)
+  or enter the commands in this guide into an Amazon EC2 [user data
+  script](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html),
+  Google Compute Engine [startup
+  script](https://cloud.google.com/compute/docs/instances/startup-scripts),
+  or similar.
 
-If you are curious to see how Teleport works, you can get started by [spinning
-up a demo cluster](./get-started.mdx) on a Linux server. After seeing how your
-demo Teleport cluster lets you securely access a server and play back your SSH
-sessions, you can configure RBAC, add resources, and protect your home lab with
-Teleport.
+  <Admonition type="tip" title="Quick demo environments">
 
-You can also get started right away with a production-ready Teleport cluster.
-[Sign up for a free trial](https://goteleport.com/signup/) of Teleport
-Team.
+  For a quick demo environment you can use to follow this guide, consider
+  installing our DigitalOcean 1-Click droplet. View the installation page on
+  [DigitalOcean
+  Marketplace](https://marketplace.digitalocean.com/apps/teleport). Once your
+  droplet is ready, SSH into the droplet and follow the configuration wizard.
 
-Once you are ready to learn more about Teleport, read our [Core Concepts
-guide](./core-concepts.mdx), which introduces the components of a Teleport
-cluster. You can refer to this glossary as you continue through the
-documentation.
+  </Admonition>
 
-## Choose an edition
+- A two-factor authenticator app such as [Authy](https://authy.com/download/),
+  [Google Authenticator](https://www.google.com/landing/2step/), or [Microsoft
+  Authenticator](https://www.microsoft.com/en-us/account/authenticator)
 
-After trying out Teleport, you are ready to deploy a cluster to your
-infrastructure. Teleport has four editions: 
+You must also have one of the following:
+- A registered domain name.
+- An authoritative DNS nameserver managed by your organization, plus an existing
+  certificate authority. If using this approach, ensure that your browser is
+  configured to use your organization's nameserver.
```

Comment:
"plus an existing certificate authority" - Is this required when using LE?

Reply:
LE would be the first option. I've bolded "one" to emphasize that it's an exclusive OR.